js/course_settings_disable_course_conclude.js

```diff
@@ -48,11 +48,16 @@ function sortRoleDropdown(){
     roleSelect.html(optList);
 }
 
+function disableCopyCourseButton() {
+    $('a.copy_course_link').addClass('disabled');
+}
+
 function initHUGlobal() {
     var reCourseSettingsPage = /courses\/.+?\/settings/;
     var windowUrl = window.location.pathname;
     var onCourseSettingsPage = (windowUrl.search(reCourseSettingsPage) != -1);
     if (onCourseSettingsPage) {
+        disableCopyCourseButton();
```

Comment:
Along the lines of having a check for the conclude button(isConcludeButtonPresent on line 61), maybe a good idea to  have a check for the copy button before invoking the disable function. I cant recollect the details, but we may have had some JS errors in some cases.